src/app/tasks/shared/task.model.ts

```diff
@@ -0,0 +1,21 @@
+export interface Task {
+  id: string;
+  associatedLink: string;
+  context: string;
+  type: string;
+  region: string;
+  district: string;
+  priority: string;
+  startDate: string;
```

Comment:
Shouldn't these be of type: Date ?

Reply:
it's just example mock data, it has no impact